cmd/dep/main.go

```diff
@@ -21,6 +21,15 @@ import (
 	"github.com/golang/dep/internal/fs"
 )
 
+const constraintSyntaxDescription = `Version contraint syntax:
+
+"dep ensure" uses an external semver library
+(https://github.com/Masterminds/semver) to interpret the
+version constraints you specify in the manifest.
```

Comment:
Let's make it:
```
dep uses an external semver library (https://github.com/Masterminds/semver) to
interpret the version constraints specified in the manifest.
```
because it's not just about ensure, but dep as a whole.